### ISSUE TITLE
4.1.2 Bonus to the empty chair

### ISSUE THREAD

**Issue by kclimenhaga** [gryphonshafer/Quizzing-Rule-Book#163]
Currently, when an assigned seat bonus goes to an empty seat, the team forfeits the bonus. Sometimes, this seat is empty due to a quizzer erring out, in which case it makes sense for the team to forfeit the bonus. However, sometimes it is empty due to the team only having three members. If that is the case, could it be adjusted in some way? For example, change the following under 4.1.2.1:
_If no quizzer of the team that did not err on the toss-up is seated in chair 2, then the bonus is "for an empty seat" and considered a no jump._

To the following:
_If the quizzer to whom the bonus would have been assigned has erred-out or fouled-out, then the bonus is "for an empty seat" and considered a no jump. If no quizzer was seated on the chair to which the bonus is assigned, it is assigned to the quizzer to the left of the empty chair. If the empty seat is seat 1, the bonus is assigned to seat 4._

I know the wording is awkward, so I'm open to suggestions! Context for this proposal: at Canadian Nationals, there was a three-person team. They had about six bonuses go to the empty chair despite no one being erred out.

**Comment by 24RMiller** [comment]
I agree with your point that teams with less than 4 quizzers should not have to forfeit questions. However, my problem with your suggestion is that it allows teams to intentionally set their lineups so their best quizzer is responsible for 2 seats. It could even be possible that a team with 4 or 5 quizzers would intentionally sit quizzers so their more knowledgeable quizzers would be responsible for more of the bonuses. At this point it would defeat the point of seat bonuses, which is to reward deep teams.

My counter-proposal is this: 

If the seat to whom the bonus would have been assigned to is empty, the bonus will go to the seat of the quizzer who errored the question prior to the toss-up. If that seat is also empty, or it is the same seat the bonus was previously given to it, the question is forfeited.

This would still reward teams with deep teams while also not punishing smaller teams. If a team had 3 quizzers, they would now have a 1/16 chance of losing the bonus(assuming all quizzers on the other teams are equal in skill) compared to a 1/4 chance previously. In this case the Canadian team may have lost 1 or 2 bonuses, not 6. 

Additionally, a rule could be added:
If the seat to whom the bonus was assigned to was occupied at any time during the quiz, and was vacated for any reason, with the exception of a quiz out, the team automatically loses the bonus

Definitely could be a few wording changes, but I thought I would throw this out there as a second option.

**Comment by ARMediting** [comment]
I've been the assistant coach for two three-person teams at Internationals, and yeah we lost some bonuses, but strategic seat positions helped some. At the time it felt like reaping the consequences of not recruiting well enough as a district, but that's not fair to the quizzers who did come. So I could be open to this.

Using the previous error's seat is an intriguing suggestion. It still feels a bit arbitrary, but it may be the best option. I'd change the language to "If the seat the bonus would have been assigned to has been empty since the beginning of the quiz..." and not worry about a sentence about the seat being vacated. Would we want to have the same or different rule for a hypothetical team starting a quiz minus one or more attending quizzers (the reasons for which could range from illness to disciplinary measures)?

**Comment by jswingle** [comment]
The suggestions are intriguing, but I would prefer to not have a complex set of rules defining who can get the bonus or not. I think jumping bonuses at district competitions where 3-person teams are more likely is the way to go... whereas at Internationals it is rare to see any 3-person teams and the material knowledge is higher, making jumping bonuses too easy. I don't see the current rules as needing a change.

**Comment by josiah-leinbach** [comment]
An additional hurdle for any proposal on this issue would be to differentiate between a three-person team and a full team where quizzers have errored out, in order to preserve the rule's intended purpose. Yet all the wordings coming to mind seem overly complex and filled with many loopholes. So unfortunately, sympathetic as I am to giving alternative chances on otherwise skipped bonus questions, I do not see a viable solution, though I am open to any suggestions.

**Comment by jttower** [comment]
I agree with Jeremy that, although 3-person teams at Internationals/Nationals does happen, it is rare.  I don't think we need a change.
